packages/manager/apps/catalog-revamp/src/components/SearchBar/SearchBar.scss

```diff
@@ -0,0 +1,18 @@
+.searchbar-container {
+  margin: 20px;
+  float: right;
+
+  .searchbar {
+    float: right;
+
+    .search-button {
+      position: absolute;
+      margin-left: -10px;
```

Comment:
avoid using float and negative margin when possible : make it less maintainable and harder to refactor with tailwind